code/datums/soundtrack.dm

```diff
@@ -0,0 +1,29 @@
+GLOBAL_LIST_EMPTY(soundtrack_this_round) // A running list of soundtrack songs that have played this round, for credits entries
+
+/datum/soundtrack_song
+	var/title
+	var/artist
+	var/url
+	var/album
+	var/file
```

Comment:
This should probably have a license value as well.

Reply:
nah the files themselves are licensed and I also don't think we would take anything other than CC BY SA directly into the codebase